get-to-know-security-manager/microsoft-teams-screen.md

```diff
@@ -13,18 +13,42 @@ To start exploring features and reports, navigate to the Microsoft Teams screen
 
 Available management actions for Microsoft Teams are contextually related to the selected object, which means users can click on or hover over something to get a contextual menu that offers actions related to the selected item.
 
-## Available actions
+## All Teams report
+### All Teams Tab
+General overview of the Teams on your tenant. The report will show you the team's Display Name, privacy level, site URL, number of: channels, team members, owners, guests and the date when the team was created. 
 
-* **Remove a connection to a tenant** – Remove connections to all associated site collections and Office 365 groups from SysKit Security Manager.
-* **Connect** – Discover and connect to Office 365 tenant.
+__Contextual actions:__
 * **Clone Team** – – Create a team using an existing team as a template.
-* **Update Team Settings** – Update the properties of the selected team.
-* **Remove Team** – Remove the selected team from SysKit Security Manager.
-* **Delete Team** – Delete the selected team from your tenant.
+* **Update  Settings** – Update the properties of the selected team.
+* **Remove** – Remove the selected team from SysKit Security Manager.
+* **Delete** – Delete the selected team from your tenant.
+
+### Global Settings tab
+On this report, you can see the set values for all the Team settings. They are grouped in 4 sections: Messaging, External Access, Guest Access, Teams Settings
+### Audit tab
+to do igor
+
+## Single Team report
+
+__Global actions:__
 * **Add Owner** – Add owner to an Office 365 group. User availability in the Teams app depends on Team settings and Microsoft user synchronization between Office 365 and Microsoft Teams; this can take up to 48 hours.
 * **Add Member** – Add member to an Office 365 group. User availability in the Teams app depends on Team settings and Microsoft user synchronization between Office 365 and Microsoft Teams; this can take up to 48 hours.
-* **Remove Owner** – Remove the selected owner from the team.
-* **Remove Member** – Remove the selected member from the team.
 * **Go to Group** – Open the associated Office 365 group for a given team.
 * **Go to Site Collection** – Open the associated site collection for a given team.
 
+### Team Members tab
+A detailed overview of a single Microsoft Team, that is, a list of all the team members. The report will show you the users Display name, email, username and role.
+
+__Contextual actions:__
+* **Remove** – Remove the selected user from the team.
+
+### Channels tab
+On this report you can see a list of all the channels in one Team. We provide you with the channels Display name, description, URL and email
```

Comment:
On this report**,** you can see a list of all the channels in one Team. We provide you with the channels Display name, description, URL**,** and email**.**